src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java

```diff
@@ -216,8 +244,12 @@ public String getHppVariantOverride() {
         return hppVariantOverride;
     }
 
-    public int getPendingPaymentExpirationPeriodInDays() {
-        return pendingPaymentExpirationPeriodInDays;
+    public int getPendingPaymentExpirationPeriod() {
```

Comment:
To avoid any confusion, should we return a `org.joda.time.Period` instead? That way, we could support things like `PT1H30M` in the config?